### PR TITLE
[flutter_tools] remove handling of error that is fixed

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

The linked error has been fixed, removed work around. Updates the test cases to only test for exit instead of error message

https://github.com/flutter/flutter/issues/33050

### CODE REVIEW

**Review by blasten — APPROVED** [review]

File: packages/flutter_tools/lib/src/run_cold.dart [hunk 1]
```diff
@@ -132,15 +132,7 @@ class ColdRunner extends ResidentRunner {
...
-    } on Exception catch (error) {
-      globals.printError('Error connecting to the service protocol: $error');
-      // https://github.com/flutter/flutter/issues/33050
-      // TODO(blasten): Remove this check once https://issuetracker.google.com/issues/132325318 has been fixed.
-      if (await hasDeviceRunningAndroidQ(flutterDevices) &&
-          error.toString().contains(kAndroidQHttpConnectionClosedExp)) {
-        globals.printStatus('🔨 If you are using an emulator running Android Q Beta, consider using an emulator running API level 29 or lower.');
-        globals.printStatus('Learn more about the status of this issue on https://issuetracker.google.com/issues/132325318');
-      }
+    } on Exception {
```
Comment (christopherfujino) [inline packages/flutter_tools/lib/src/run_cold.dart:135]: do we not want to print any error?

Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/run_cold.dart:135]: Ahh good catch- I deleted too much here

Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/run_cold.dart:135]: ha ha, catch

Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/run_cold.dart]: Fixed

**Review by christopherfujino — APPROVED** [review]
LGTM

**Pull request merged**